src/apps/lwaftr/lwaftr.lua

```diff
@@ -243,7 +200,8 @@ local function binding_lookup_ipv4(lwstate, ipv4_ip, port)
       print(lwdebug.format_ipv4(ipv4_ip), 'port: ', port, string.format("%x", port))
       lwdebug.pp(lwstate.binding_table)
    end
-   local psid = lwstate.psid_info_map:lookup_psid(ipv4_ip, port)
+   local host_endian_ipv4 = C.htonl(ipv4_ip)
```

Comment:
This is true!  I was just thinking of htonl as a byte-swapping operator.  I agree it would make more sense to call ntohl.  Actually we should be avoiding calling out to C here and using something like core.lib.ntohl but I wanted a uint32, not an int32, so I called to C.  Anyway will change to call ntohl.